exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/union/UnionAllRecordBatch.java

```diff
@@ -335,7 +343,8 @@ public IterOutcome nextBatch() throws SchemaChangeException {
             return iterRight;
 
           default:
-            throw new IllegalStateException(String.format("Unknown state %s.", iterRight));
+            throw new IllegalStateException(
```

Comment:
Hmm.  It's been a while, so I don't recall exactly.  It looks like I wrapped it to fit within 80 columns since most of the code and comments just above fit within that width.  (Yes, now I see that the next throw statement below is also wider.)